tests/console/yast2_lan_hostname.pm

```diff
@@ -26,7 +27,8 @@ sub hostname_via_dhcp {
     $cmd{home}             = 'home';
     $cmd{spc}              = 'spc';
 
-    type_string "yast2 lan\n";
+    y2logsstep::yast2_console_exec(yast2_module => 'lan');
```

Comment:
I'm not sure if you actually need an import if you call method with class name. So I guess we can remove it, otherwise I would recommend to import just a single method to save some memory (which won't help much as we will load that perl module anyway cause of other tests).

Reply:
Nice point, let me check!

Reply:
I have commented the import statement and it seems to work. I will clean the modules.